setup.py

```diff
@@ -1,17 +1,16 @@
 #!/usr/bin/env python
 import sys
-from setuptools import setup
+from setuptools import setup, Distribution
 from setuptools.command.test import test as TestCommand
 from sys import platform
 from platform import architecture
-import shutil
 
 if platform == 'win32' and architecture()[0] == '32bit':
-    shutil.copy2('win/sndfile32.dll', 'win/sndfile.dll')
-    sndfile = [('', ['win/sndfile.dll', 'win/sndfile_license'])]
+    sndfile = [('lib', ['win32/libsndfile.dll', 'win32/sndfile_license'])]
```

Comment:
`''` is the Python root, not the PySoundFile root. It would be the only binary in that path. At least in `lib`, it is among other libraries.